Project.toml

```diff
@@ -20,4 +19,4 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Test"]
+test = ["Test"]
```

Comment:
How is it that there is no newline at the end of the file?

Reply:
This is automatically generated, I don't know. I can add it manually